sdk/src/invariant.ts

```diff
@@ -399,6 +506,12 @@ export class Invariant {
     ) as Promise<RemovePositionTxResult>
   }
 
+  claimFeeTx(index: bigint) {
+    return getTx(this.contract, this.gasLimit, this.storageDepositLimit, 0n, InvariantTx.ClaimFee, [
+      index
+    ])
+  }
+
   async claimFee(account: IKeyringPair, index: bigint, block: boolean = true): Promise<TxResult> {
     return sendTx(
```

Comment:
Rename it to `sendAndSignTx` as this function doing both. Then you use name `sendTx` to function that only send signed transaction